R/class-workbook-wrappers.R

```diff
@@ -1796,20 +1787,20 @@ wb_remove_named_region <- function(wb, sheet = current_sheet(), name = NULL) {
 
 # filters -----------------------------------------------------------------
 
-#' Add column filters
+#' Add/remove column filters in a worksheet
 #'
-#' Add excel column filters to a worksheet
+#' Add or remove excel column filters to a worksheet
 #'
+#' Adds filters to worksheet columns, same as filter parameters in write_data.
+#' [wb_add_data_table()] automatically adds filters to first row of a table.
+#' NOTE Can only have a single filter per worksheet unless using tables.
 #' @param wb A workbook object
-#' @param sheet A name or index of a worksheet
+#' @param sheet
+#'   * In `wb_add_filter()` A single name or index of a worksheet.
+#'   * In `wb_remove_filter()`, a vector of names/indices of worksheet
```

Comment:
Thanks for the PR! Looking good, minor nitpicking: This should be the same as everywhere else. The mistake was in the never updated documentation.

Reply:
I have changed it, since I realized the .Rd file didn't really render as expected. 

But do you want me to update the docs for other `wb_remove_*` functions? As of now, I haven't touched the documentation for other `wb_remove_*` functions

Reply:
Guess wherever we use a sheet waiver the same text should work. It just looks like the arguments take different input.

Reply:
Ok, still unsure of how to implement it.